### DISCUSSION TITLE
I want to use tauri v1 to write a program that can run on the mips architecture. Is there any solution?

### DISCUSSION THREAD

**Discussion by elonehoo** [tauri-apps/tauri#11698] [Q&A]
I wonder if anyone has compiled a tauri v1 project under the mips architecture.

**Comment by opeolluwa** [comment]
My guess, since Taurus is also a Rust application, would you try to install the selected target and build for it

**Comment by elonehoo** [comment]
I failed when installing rustup

**Reply by elonehoo** [reply]
I don't know how to describe my problem. I want to know if anyone in the community has tried it and can give me some ideas.

**Reply by opeolluwa** [reply]
No I don't
